nested-spire/README.md

```diff
@@ -0,0 +1,226 @@
+       -downstream \
+       -ttl 3600
+```
+
+The `-parentID` flag contains the SPIFFE ID of the `root-agent`. The SPIFFE ID of the `root-agent` is created by the [x509pop Node Attestor](https://github.com/spiffe/spire/blob/master/doc/plugin_server_nodeattestor_x509pop.md) plugin which defines the SPIFFE ID as `spiffe://<trust domain>/spire/agent/x509pop/<fingerprint>`. A `fingerprint()` function in the shell script calculates the SHA1 fingerprint of the certificate.
+The other point to highlight is the `-downstream` option. This option, when set, indicates that the entry describes a downstream SPIRE Server.
+
+## Run the Scenario
+
+Use the `set-env.sh` script to run all the services that make up the scenario. The script starts the `root`, `nestedA`, and `nestedB` services with the configuration options described earlier.
+
+Ensure that the current working directory is `.../spire-tutorials/nested-spire` and run:
+
+```console
+    bash scripts/set-env.sh
+```
+
+Once the script is completed, in another terminal run the following command to review the logs from all the services:
+
+```console
+    docker-compose logs -f -t
+```
+
+
+# Part 2: Test the Deployments
+
+Now that the SPIRE deployments are ready, let's test the scenario that we've configured.
+
+## Create Workload Registration Entries
+
+To test the scenario we create two workload registration entries, one entry for each nested SPIRE Server (`nestedA` and `nestedB`). The goal of the test is to demonstrate that SVIDs created in a nested configuration are valid in the entire trust domain, not only in the scope of the SPIRE Server that originated the SVID. The following commands demonstrate the command line options we'll use to create the two workload registration entries, but you can run these commands using the `create-workload-registration-entries.sh` script shown a few lines below.
+
+```console
+   # Workload for nestedA deployment
+   docker-compose exec -T nestedA-server \
+       /opt/spire/bin/spire-server entry create \
+       -parentID "spiffe://example.org/spire/agent/x509pop/$(fingerprint nestedA/agent/agent.crt.pem)" \
+       -spiffeID "spiffe://example.org/nestedA/workload" \
+       -selector "unix:uid:1001" \
+       -ttl 0
+
+   # Workload for nestedB deployment
+   docker-compose exec -T nestedB-server \
+       /opt/spire/bin/spire-server entry create \
+       -parentID "spiffe://example.org/spire/agent/x509pop/$(fingerprint nestedB/agent/agent.crt.pem)" \
+       -spiffeID "spiffe://example.org/nestedB/workload" \
+       -selector "unix:uid:1001" \
+       -ttl 0
+```
+
+The examples use the `$(fingerprint nestedA/agent/agent.crt.pem)` form again to show that the `-parentID` flag specifies the SPIFFE ID of the nested SPIRE Agent. The TTL flag value of zero indicates that the default value of 3600 seconds will be used. Finally, in both cases the unix selector assigns the SPIFFE ID to any process with a uid of 1001.
```

Comment:
it can be read than we use the same figerprint for both that is not true, maybe you can clarify it in another way?

Reply:
Andrés' wording may not be too confusing to users, but here are some ideas:
"examples use the $(fingerprint nested[A|B]/agent/agent.crt.pem) form again"  ?
OR 
"examples use the fingerprint() function to substitute the certificate fingerprint part of the nestedA and nestedB SPIFFE IDs when setting those SPIFFE IDs as the parent ID of each workload"  ?

Reply:
What about this?
_The examples use the `fingerprint path/to/nested-agent-cert` form again to show that..._

The idea behind the `fingerprint` function is mentioned before in the tutorial, there is no need to repeat that. The goal of this paragraph is to highlight the use of the nested agents as parent id of the new workload entries.